OWNERS

```diff
@@ -17,3 +16,6 @@ reviewers:
   - kpostoffice
   - sutaakar
   - tedhtchang
+
+emeritus_approvers:
+  - anishasthana
```

Comment:
Should we add a date?
[Kubernetes Contributors Reference](https://www.kubernetes.dev/docs/guide/owners/#emeritus:~:text=emeritus_approvers%3A,2019%2D01%2D05)

Reply:
Done